src/components/Flyout/Flyout.spec.tsx

```diff
@@ -23,8 +29,23 @@ const Component: FC<Pick<FlyoutProps, "onClick" | "onClose" | "badges">> = ({ on
             badges={badges}
             onClick={onClick}
             onClose={onClose}
+            legacyFooter={legacyFooter}
         >
             <TextInput placeholder="placeholder" />
+            {legacyFooter === false && (
+                <FlyoutFooter
+                    primaryButton={
+                        <Button key="add" style={ButtonStyle.Primary} onClick={cy.stub().as("onClickStub")}>
+                            Add
+                        </Button>
+                    }
+                    secondaryButton={
+                        <Button key="cancel" style={ButtonStyle.Secondary} onClick={cy.stub().as("onCloseStub")}>
+                            Cancel
+                        </Button>
```

Comment:
Naming primary/secondary is misleading because it's already in use for buttonStyle. 
I suggest to use [TS tuples](https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple) like we do it in [tooltip](https://github.com/Frontify/arcade/blob/fb75b2a37944e3e9ea1455c2df546340bc5c4cfd/src/components/Tooltip/Tooltip.tsx#L39).